src/modules/dashboard/components/CreateUserWizard/StepConfirmTransaction.tsx

```diff
@@ -28,15 +27,27 @@ const displayName = 'dashboard.CreateUserWizard.StepConfirmTransaction';
 const StepConfirmTransaction = () => {
   const { username } = useLoggedInUser();
   const transactionGroups = useSelector(groupedTransactions);
-  const { location } = useHistory<{ colonyURL?: string }>();
+  const { location } = useHistory<{
+    colonyURL?: string;
+    redirectTo?: string;
+  }>();
 
   if (username) {
     const { colonyURL } = location.state;
     if (colonyURL) {
       return <Redirect to={colonyURL} />;
     }
 
-    return <Redirect to={LANDING_PAGE_ROUTE} />;
+    return (
+      <Redirect
+        to={{
+          pathname: location.state.redirectTo,
```

Comment:
What if there's no `redirectTo` in the location state, where will this redirect to ?

Reply:
When `redirectTo` is `undefined`, 2 things can happen:

1. You get redirected to `/connect` if you don't have a wallet connected.
2. If you are connected and have a username, you get redirected to the landing page but I think that's a side effect of the code we have in `WalletRequiredRoute`

So, I'm gonna add the landing page route as default, just to be sure

Reply:
Great